### DISCUSSION TITLE
Select in a DataGrid

### DISCUSSION THREAD

**Discussion by RamadaniLeotrim** [microsoft/fluentui-blazor#2138] [Q&A] [answered]
I wanted to put a Select inside a TemplateColumn of a Data Grid, but the menu is not visible when clicked on the Select. 
Is the Select component not supposed to work inside a Data Grid?

**Comment by vnbaaij** [comment, marked as answer]
See #217 for origin of the issue and possible workarounds.

I'll share the same advice as I did there and elsewhere...do **not** use the grid as a big data editing tool. It is meant to **display** data efficiently. If you want edit capabilities, add a button and use that to pull up a panel or dialog or form or whatever to edit the data for a specific row.

**Reply by leetheman** [reply]
How is this meant to compete with Telerik/DevExpress if we cannot have in-line data editing capabilities?  Or is the purpose of this library for non-CRUD applications?

**Reply by vnbaaij** [reply]
What a negative way to start a question /discussion.

It is lika I say in the post marked as an answer. Primary focus is to display data. You can add row click, double click button or whatever to start an edit action on that data item.
If that is not good enough for you then this is most probably not your library. Which is perfectly fine by me.
